webapp/channels/src/plugins/rhs_plugin/rhs_plugin.tsx

```diff
@@ -13,26 +13,24 @@ export type Props = {
     title: React.ReactNode;
 }
 
-export default class RhsPlugin extends React.PureComponent<Props> {
-    render() {
-        return (
-            <div
-                id='rhsContainer'
-                className='sidebar-right__body'
-            >
-                <SearchResultsHeader>
-                    {this.props.title}
-                </SearchResultsHeader>
-                {
-                    this.props.showPluggable &&
-                    <>
-                        <Pluggable
-                            pluggableName='RightHandSidebarComponent'
-                            pluggableId={this.props.pluggableId}
-                        />
-                    </>
-                }
-            </div>
-        );
-    }
+const RhsPlugin: React.FC<Props> = ({ showPluggable, pluggableId, title }) => {
```

Comment:
The explicit typing is not needed.

Reply:
Yes. You can leave it just like this:
```suggestion
const RhsPlugin = ({ showPluggable, pluggableId, title }) => {
```